src/entry_type.rs

```diff
@@ -1,4 +1,8 @@
 // See https://en.wikipedia.org/wiki/Tar_%28computing%29#UStar_format
+// and
+// https://www-01.ibm.com/support/knowledgecenter/SSLTBW_1.13.0/
+// com.ibm.zos.r13.bpxa500/pxarchfm.htm
+
```

Comment:
I propose linking to http://pubs.opengroup.org/onlinepubs/9699919799/utilities/pax.html instead as a more accessible (and complete) page.